x/observer/types/chain_params.go

```diff
@@ -155,6 +155,24 @@ func ValidateChainParams(params *ChainParams) error {
 	return nil
 }
 
+// InboundConfirmationFast returns the fast number of confirmation for inbound observation.
+// It falls back to safe confirmation count if fast mode is disabled.
+func (cp ChainParams) InboundConfirmationFast() uint64 {
```

Comment:
Can we maybe have `InboundConfirmationSafe` and outbound for consistency in the API?

Reply:
Added both `InboundConfirmationSafe` and `OutboundConfirmationSafe`, all calls are now made through APIs: https://github.com/zeta-chain/node/pull/3496/commits/e7ee4445aa9d635ee6a08482d65ddcdadbef8d91